lstchain/io/io.py

```diff
@@ -443,6 +445,239 @@ def read_array_info(filename):
     return array_info
 
 
+def read_optic(filename, telescope_name):
```

Comment:
read_optics

Reply:
Probably this got in here by a mistake of mine? This was introduced in a different PR which I probably merged for tests, and then pushed by mistake.